util/filesystem.go

```diff
@@ -43,20 +43,38 @@ var dfHeaderPattern = regexp.MustCompile(
 
 var logger = logging.GetLogger("util.filesystem")
 
+var dfOpt = "-Pkl"
+
+func init() {
+	// Some `df` command such as busybox does not have `-P` option.
+	tio := &timeout.Timeout{
+		Cmd:       exec.Command("df", dfOpt),
+		Duration:  3 * time.Second,
+		KillAfter: 1 * time.Second,
+	}
+	exitSt, _, stderr, err := tio.Run()
+	if err == nil && exitSt.Code != 0 && strings.Contains(stderr, "df: invalid option -- P") {
```

Comment:
Maybe this will be broken when `dfOpt` is reordered like `-lPk` and/or error message of busybox changes, and I'm afraid that we'll miss the breakage because we don't have CI for busybox

Reply:
I do not plan to deal with busybox very seriously, so we do not have to worry about that. For now, I will remove the "P" here.